requirements.txt

```diff
@@ -20,3 +20,4 @@ flake8-tuple>=0.4.0
 isort==4.3.21
 mypy==0.740
 bump2version>=1.0.0
+yaml==5.3.1
```

Comment:
blocking: I believe we already have a YAML library available as a dependency: https://github.com/determined-ai/determined/blob/master/common/setup.py#L22. See the example of it's use here: https://github.com/determined-ai/determined/blob/58e439083e22ec742f31d5adb4dde511859b7875/cli/determined_cli/experiment.py#L214